assets/src/components/event-attendees/attendee-list-item.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External imports
+ */
+import { isModelEntityOfModel } from '@eventespresso/validators';
+import warning from 'warning';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal imports
+ */
+import { AvatarImage } from '../ui/image';
+
+const AttendeeListItem = ( {
```

Comment:
This was added to the components folder (and thus exported with `@eventespresso/components`) because its not specific to a block.  It's feasible that it could be used elsewhere in a ui (and components are "context" agnostic).